### DISCUSSION TITLE
Support for Catalyst apps?

### DISCUSSION THREAD

**Discussion by adamwulf** [dataFude-app/dataFude#8] [Ideas]
Great app! very interesting and I can see how this could be very useful to my debugging/dev workflow.

It seems to only work for iOS simulators - any plans on supporting Mac apps, specifically catalyst?

**Comment by icanzilb** [comment]
Yes, "dataTile for Simulator" streams the device logs from the simulator so it leverages Apple's unified logging. This constraint is mostly imposed by Apple's own sandbox so the app can ship to the App Store.

My own build locally isn't sandboxed so it works with either the unified log or the app's output from prints.

In case, the launch to the App Store shows there's interest in such a tool from people willing to pay for devtools, the next step would definitely be launching a non-sandbox alternative for debugging Catalyst and Mac apps.

I hope that makes sense?

**Comment by adamwulf** [comment]
yep makes sense :) put me on the list for the non-sandboxed version :)

**Reply by icanzilb** [reply]
Sure, it's the same list as for this beta, no worries
